### PR TITLE
don't treat pycache as a module in test_module_state

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: tests/test_module_state.py [hunk 1]
```diff
@@ -19,7 +19,7 @@
 def find_all_modules():
     for module in MODULES_PATH.listdir(sort=True):
         module = module.basename.replace('.py', '')
-        if module != '__init__':
```
Comment (mdellweg) [inline tests/test_module_state.py]: `if module.basename.endswith(".py"):` ?

Comment (mdellweg) [reply tests/test_module_state.py]: Oh, that does not work with `__init__.py`. Darn.

Comment (evgeni) [reply tests/test_module_state.py]: that would also work, yes

Comment (mdellweg) [reply tests/test_module_state.py]: What about a regexp match:
`r"\([^-].*\)\.py"` and the first matching group is what you want anyway.

**Comment by evgeni** [comment]
not sure why we didn't see that before, maybe something changed how py.path works?

**Pull request merged**